support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -24,15 +25,29 @@ export function CheckoutNudgeContainer({
 
 	const [displayNudge, setDisplayNudge] = useState(true);
 
-	const currencyGlyph = glyph(detect(countryGroupId));
-	const minAmount = config[countryGroupId]['ANNUAL'].min;
+	const nudgeMinVariantA = useContributionsSelector(
+		isUserInAbVariant('nudgeMinAmountsTest', 'variantA'),
+	);
+	const nudgeMinVariantB = useContributionsSelector(
+		isUserInAbVariant('nudgeMinAmountsTest', 'variantB'),
+	);
 
+	const currencyGlyph = glyph(detect(countryGroupId));
+	const minAmount = getConfigAbTestMin(countryGroupId, 'ANNUAL', {
+		variantA: nudgeMinVariantA,
+		variantB: nudgeMinVariantB,
+	});
+	const roundToNearest = 10;
+	const minAmountRounded =
```

Comment:
Is this say the `weeklyMinAmount`?

Reply:
Yes thats a better variable name, think that describe what the variable is for rather than ho wit works. Changed